sample_data/groundwater_protection_zones/legend_entry.json

```diff
@@ -1,9 +1,9 @@
 [
   {
     "legend_text": {
-      "fr": "Pollué, investigation nécessaire",
-      "de": "Belastet, untersuchungsbedürftig",
-      "it": "Inquinato, è necessario procedere a un?indagine"
+      "fr": "Zone de captage (zone S1)",
```

Comment:
Same here: not directly linked to the PR, but I thought our sample data could be a little more realistic :)